cmd/network/evpn-utils.go

```diff
@@ -52,7 +53,7 @@ func PrintBP(bp *pb.BridgePort) {
 	log.Println("name:", bp.GetName())
 	log.Println("status:", bp.GetStatus().GetOperStatus().String())
 	log.Println("ptype:", bp.GetSpec().GetPtype())
-	log.Println("mac:", bp.GetSpec().GetMacAddress())
+	log.Println("mac:", net.HardwareAddr(bp.GetSpec().GetMacAddress()).String())
```

Comment:
are changes in this file related to the PR goal? If they are not TLS related, they have to be a part of a separate PR

Reply:
reverted back to previous commit.